src/dmd/backend/cgcod.d

```diff
@@ -3467,4 +3474,32 @@ void andregcon(con_t *pregconsave)
     regcon.cse.mops &= regcon.cse.mval;
 }
 
+
+/**********************************************
+ * Disassemble the code instruction bytes
+ * Params:
+ *    code = array of instruction bytes
+ */
+@trusted
```

Comment:
I think this is concerning. This is basically `@trusted` 4k of lines?

Reply:
On the backend, I added @trusted to everything, as a first step towards using @safe. It's much harder to find the @safe code without doing that.